src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,30 @@
+package seedu.addressbook.data.person;
+
+
+public class Contact {
+
+    public final String value;
```

Comment:
Can we prevent the exposing of `value` outside the class? Will using `protected` scope help us to achieve the same intended functionality?